src/ui/pages/IdentifierDetails/components/IdentifierContent.tsx

```diff
@@ -33,7 +33,7 @@ const IdentifierContent = ({
       {cardData.di !== "" && (
         <CardDetailsBlock title={i18n.t("identifiers.details.delegator.title")}>
           <CardDetailsItem
-            info={cardData.di}
+            info={cardData.di as string}
```

Comment:
I guess we don't need this change if we set the statement above to `cardData.di !== undefined` right?